src/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+
+const { randomNumbers } = require('./randomNumber.js');
+const { terminal } = require('./terminal.js');
+const { countBullsAndCows } = require('./countBullsAndCows.js');
+
+let attemptCnt = 1;
+
+const userAnswer = () => {
+  terminal.question('Please enter 4 different digits\n', (numFromUser) => {
+    if (numFromUser === randomNumbers) {
+      terminal.write(`You guessed the number ${randomNumbers} \n
+        after ${attemptCnt} attempt(s)`);
+      terminal.close();
+    } else if (numFromUser.length === 4) {
```

Comment:
You need to check if all digits are different.
![image](https://user-images.githubusercontent.com/102293862/195652630-8a5cf00c-e2ca-480b-aa38-53955032db0a.png)